cobbler/utils.py

```diff
@@ -1597,20 +1980,54 @@ def subprocess_sp(logger, cmd, shell=True, input=None):
 
 
 def subprocess_call(logger, cmd, shell=True, input=None):
+    """
+    A simple subprocess call with no output capturing.
+
+    :param logger: The logger to audit the action with.
+    :param cmd: The command to execute.
+    :param shell: Whether to use a shell or not for the execution of the commmand.
+    :type shell: bool
+    :param input: If there is any input needed for that command to stdin.
+    :return: The return code of the process
+    """
     data, rc = subprocess_sp(logger, cmd, shell=shell, input=input)
     return rc
 
 
 def subprocess_get(logger, cmd, shell=True, input=None):
+    """
+    A simple subprocess call with no return code capturing.
+
+    :param logger: The logger to audit the action with.
+    :param cmd: The command to execute.
+    :param shell: Whether to use a shell or not for the execution of the commmand.
+    :type shell: bool
+    :param input: If there is any input needed for that command to stdin.
+    :return: The data which the subprocess returns.
+    """
```

Comment:
docstring 3 times longer than the method and C&P from above...
Better add a reference to the other method or leave docstrings outs